src/components/EventCard/popup/deletePopUp.tsx

```diff
@@ -1,34 +1,32 @@
 import React, { useState } from 'react';
+import DeleteSharpIcon from '@mui/icons-material/DeleteSharp';
+import { BiztechEvent } from '@/types/biztechEvent';
 
 
 interface Props {
     setIsDelete: React.Dispatch<React.SetStateAction<boolean>>,
-    bizEvent: string
+    bizEvent: BiztechEvent | null | undefined,
```

Comment:
I think this was giving me a type mismatch error when I tried to pass it as a prop that I didn't understand initially. I guess since the cards are only mapped if the data is inputted we wouldn't have to worry about it being null? I could define a 'defaultBiztechEvent' and have that as the initial state if nothing is passed in, or do you think I should just have it as 'BiztechEvent | null" instead?

Reply:
Thanks for the explanation Marcus - the `BiztechEvent | null` solution seems fine for now

Once we have the backend integrated, using only `BiztechEvent` as the type makes the most sense to me, since we shouldn't be mapping items that don't exist from the `BiztechEvent[]` array. Even if the initial state's array was empty, we wouldn't pass in nulls to the cards and pop-ups

I think the type mismatch error comes from the admin/home.tsx file, where we're expecting props to be `initialData: BiztechEvent[] | null`. I think we can leave this change for another time once the backend is integrated and see if we can phase out the `null`s, or if they're necessary with the way Next.js works